tests/functional/object/mcg/test_mcg_performance_profiles.py

```
@@ -0,0 +1,478 @@
+                "lim_cpu": "4",
+                "req_mem": "2Gi",
+                "lim_mem": "4Gi",
+                "qos": "Burstable",
+            },
+            "endpoint_count": {"min": 2, "max": 4},
+            "db_instances": 2,
+            "pv_pool": {"cpu": "1", "mem": "2Gi"},
+        },
+    }
+
+    @pytest.fixture()
```

Ruff (0.16.2)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)